modules/express/src/lightning/lightningSignerRoutes.ts

```diff
@@ -84,7 +84,7 @@ export async function handleInitLightningWallet(req: express.Request): Promise<u
     }
   );
 
-  const wallet = await coin.wallets().get({ id: walletId });
+  const wallet = await coin.wallets().get({ id: walletId, includeBalance: false });
```

Comment:
`handleCreateSignerMacaroon` too

Reply:
I think pay should have it, if we can't get the balance for some reason then that means we cant connect to the node. So i think this is valid.